### PR TITLE
Refactoring controllers

### PR DESCRIPTION
**Pull request by russellvaughan**
refactored controllers and artist sign up tests all passing now

### CODE REVIEW

File: back-end/app/models/stream.rb [hunk 1]
```diff
@@ -1,4 +1,27 @@
...
+	belongs_to :track
+	belongs_to :user
+
+	def self.weekly_usage(streams)
+		user_weekly_plays = 0
+		tracks_played=[]
+		individual_track_plays=[]
+
+    streams.each do |stream|
+			user_weekly_plays += stream.weekly_plays
```
Comment (drjparry) [inline back-end/app/models/stream.rb]: indentation

File: back-end/db/migrate/20160331100643_set_default_users.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class SetDefaultUsers < ActiveRecord::Migration
+	def self.up
+		change_column :users, :balance, :decimal, default: 0, precision: 8,  scale: 2 
+	end
+
+	def self.down
+		raise ActiveRecord::IrreversibleMigration, "can't remove default"
+	end
```
Comment (drjparry) [inline back-end/db/migrate/20160331100643_set_default_users.rb]: indentation

File: back-end/spec/models/artist_spec.rb [hunk 3]
```diff
@@ -1,5 +1,6 @@
 require 'rails_helper'
```
Comment (ghost) [inline back-end/spec/models/artist_spec.rb:2]: extra line?

File: back-end/spec/rails_helper.rb [hunk 4]
```diff
@@ -1,4 +1,4 @@
-ENV['RAILS_ENV'] ||= 'test'
+ENV['RAILS_ENV'] ||= 'development'
```
Comment (ghost) [inline back-end/spec/rails_helper.rb]: test environment?

**Comment by drjparry** [comment]
:shipit:

**Pull request merged**